.github/workflows/test.yml

```diff
@@ -4,10 +4,10 @@ on: [push, pull_request]
 
 jobs:
     build:
-        runs-on: ubuntu-latest
+        runs-on: ubuntu-20.22
```

Comment:
Change runner to `ubuntu-20.04`

`ubuntu-20.22` was disabled by github recently

Reply:
I tried changing it to ubuntu-20.04, but no luck with that version it's giving these failures, 
Do you have any other suggestions?

https://github.com/imagekit-developer/imagekit-python/actions/runs/5056431996/jobs/9073896293?pr=39